test/sharness/lib/test-lib.sh

```diff
@@ -394,6 +394,27 @@ file_size() {
     $_STAT "$1"
 }
 
+directory_size() {
```

Comment:
Why not just something like `du -sb --apparent-size`?

Reply:
1. Didn't know about it.
2. MacOS :sad:.